docs/getting-started.mdx

```diff
@@ -4,11 +4,19 @@ import HowItWorksDiagram from '../assets/getting-started-how-superface-works.svg
 
 Superface is all about you, the consumer of API capabilities. To get started, first pick a capability and its provider, install it to your App and use Superface OneSDK to make the call.
 
+:::info
+
+This guide is more detailed version of [Getting started](https://superface.ai/getting-started) guide available after registration.
```

Comment:
"available after registration"  → "available at superface.ai"

Reply:
what if we have registration elsewhere too ? :)

Reply:
Well, that makes sense but you need to have an account to access that page. If you are not logged in and click that link, you end up on login page. So how about:

```suggestion
This guide is more detailed version of [Getting started](https://superface.ai/getting-started) guide available after registration at superface.ai.
```

or:

```suggestion
This guide is more detailed version of [Getting started](https://superface.ai/getting-started) guide available with superface.ai account.
```

Reply:
I would still stick with the one I'v suggested, if they do not have an account, then sure they need to create one :)

Reply:
```suggestion
This guide is more detailed version of the in-product [Getting started](https://superface.ai/getting-started) available at superface.ai.
```